pkg/apis/core/v1alpha1/types.go

```diff
@@ -70,6 +70,7 @@ type DataVolumeSource struct {
 	Upload   *DataVolumeSourceUpload   `json:"upload,omitempty"`
 	Blank    *DataVolumeBlankImage     `json:"blank,omitempty"`
 	Imageio  *DataVolumeSourceImageIO  `json:"imageio,omitempty"`
+	VDDK     *DataVolumeSourceVDDK     `json:"vddk,omitempty"`
```

Comment:
Does this overrule @mhenriks' previous request to add them to the alpha API? https://github.com/kubevirt/containerized-data-importer/pull/1306#discussion_r457594547

It has been some time since that comment, so if it's really no longer needed I can take it out.

Reply:
Good question, let me discuss with @mhenriks since this is net new code, I don't think we need to add it in alphav1, but I might be missing some good reason it should be there.

Reply:
I discussed this, and we need to keep it in alpha as well as beta, so you can ignore this.